website/www/site/content/en/documentation/runners/flink.md

```diff
@@ -276,36 +276,48 @@ If you have a Flink `JobManager` running on your local machine you can provide `
 `flinkMaster`. Otherwise an embedded Flink cluster will be started for the job.
 {{< /paragraph >}}
 
-Starting with Beam 2.18.0, pre-built Docker images are available at Docker Hub.
-
-JobService:
+{{< paragraph class="language-portable" >}}
+Starting with Beam 2.18.0, pre-built Flink Job Service Docker images are available at Docker Hub:
 [Flink 1.8](https://hub.docker.com/r/apache/beam_flink1.8_job_server),
 [Flink 1.9](https://hub.docker.com/r/apache/beam_flink1.9_job_server).
 [Flink 1.10](https://hub.docker.com/r/apache/beam_flink1.10_job_server).
-
-Beam SDK:
-[Python 2.7](https://hub.docker.com/r/apache/beam_python2.7_sdk),
-[Python 3.5](https://hub.docker.com/r/apache/beam_python3.5_sdk),
-[Python 3.6](https://hub.docker.com/r/apache/beam_python3.6_sdk),
-[Python 3.7](https://hub.docker.com/r/apache/beam_python3.7_sdk).
+{{< /paragraph >}}
 
 To run a pipeline on an embedded Flink cluster:
 <!-- Span implictly ended -->
 
 {{< paragraph class="language-py" >}}
-1. Start the JobService endpoint: `docker run --net=host apache/beam_flink1.10_job_server:latest`
+(1) Set the runner to `FlinkRunner`.  Optionally set `environment_type` set to `LOOPBACK`. For example:
 {{< /paragraph >}}
 
-{{< paragraph class="language-py" >}}
+{{< highlight py >}}
+import apache_beam as beam
+from apache_beam.options.pipeline_options import PipelineOptions
+
+options = PipelineOptions([
+    "--runner=FlinkRunner",
+    "--environment_type=LOOPBACK"
+])
+with beam.Pipeline(options) as p:
+    ...
+{{< /highlight >}}
+
+{{< paragraph class="language-portable" >}}
+<!-- TODO(BEAM-10214): Use actual lists here and below. -->
+(1) Start the JobService endpoint: `docker run --net=host apache/beam_flink1.10_job_server:latest`
```

Comment:
This instruction appears when Python is toggled. I don't think it should.

Reply:
Fixed.